mine_extractor_refactored/config/settings.py

```
@@ -0,0 +1,111 @@
+                    "double_quote": True
+                },
+                "encoding": "utf-8"
+            },
+            "extraction_strategy": "hybrid",
+            "max_retries": 3,
+            "backoff_base_seconds": 1,
+            "backoff_max_seconds": 60,
+            "backoff_jitter": 0.1
+        }
+
+def _resolve_environment_variables(config: Dict[str, Any]) -> Dict[str, Any]:
```

Ruff (0.12.2)

Undefined name `Dict`

(F821)